### PR TITLE
Issue template: request for link to Check

### PR DESCRIPTION
**Pull request by romani**
Each time I read issue, and I have to open documentation to make decision on approval.
It would help user to review documentation before reporting issue and for us to quickly read documentation.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
+Check documentation: https://checkstyle.org/config_xxxxxx.html#XxxxxxXxxxxxXxxxxx
```
Comment (pbludov) [inline .github/ISSUE_TEMPLATE.md]: Such a link is of no use. It is better to add a link to https://checkstyle.org/checks.html where everyone can get to the desired config in one click.

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: such link will not trigger any action from user to provide direct link to Check.
we give template .... user need to fill template

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: What about  https://checkstyle.org/config_[[package name]].html#[[NameOfAffectedCheck]] ?

Comment (pbludov) [reply .github/ISSUE_TEMPLATE.md]: That's better. Now it became clear what should be filled. It is possible that advanced users will be able to fill this out correctly.

 "Name of affected check" is obvious here, but with "package name" users may have problems.

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: Should we place xxxxx as sign of "fill it" ?
Any suggestions ?

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: @strkkk please share your ideas on how to make it clear for user ?

Comment (rnveach) [reply .github/ISSUE_TEMPLATE.md]: @romani There have a been few times where people left `[[]]` as is. Would it make sense to add a statement somewhere to the template to denote `[[]]` must be replaced?

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: Please look at diff, I didnot  use `[[` in URL.

**Comment by romani** [comment]
> Based on other talking points, line 14 should remove xx.

I do not understand, please explain.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (rnveach) [reply .github/ISSUE_TEMPLATE.md]: > Please look at diff, I didnot use [[ in URL.

I was referring to discussion that wanted to use `[[` in URL.
https://github.com/checkstyle/checkstyle/pull/7327#discussion_r355099402

**Comment by rnveach** [comment]
> I do not understand, please explain.

Based on https://github.com/checkstyle/checkstyle/pull/7327#discussion_r355104154 and https://github.com/checkstyle/checkstyle/pull/7327#discussion_r355099402

Change `/var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java` to something like `/var/tmp $ java $RUN_LOCALE -jar checkstyle-[[version]]-all.jar -c config.xml YOUR_FILE.java`.
I was just going based on conversation to add more `[[`s.

**Comment by romani** [comment]
I think it is better to not use braces, but if all other think differently, I can change

**Comment by pbludov** [comment]
For me, `checkstyle-X.XX-all.jar` more obvious than `checkstyle-[[version]]-all.jar`.

**Comment by pbludov** [comment]
@romani please update example at `report_issue.html` too.

**Comment by romani** [comment]
> please update example at report_issue.html too.

done.

**Comment by romani** [comment]
@strkkk, please review.

**Comment by rnveach** [comment]
@romani Travis CI failed.

**Pull request merged**